css/at-rules/layer.json

```
@@ -18,28 +18,40 @@
             "edge": {
               "version_added": false
             },
-            "firefox": {
-              "version_added": "94",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "layout.css.cascade-layers.enabled",
-                  "value_to_set": "true"
-                }
-              ],
-              "notes": "See Firefox <a href='https://bugzil.la/1699217'>bug 1699217</a>."
-            },
-            "firefox_android": {
-              "version_added": "94",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "layout.css.cascade-layers.enabled",
-                  "value_to_set": "true"
-                }
-              ],
-              "notes": "See Firefox <a href='https://bugzil.la/1699217'>bug 1699217</a>."
-            },
+            "firefox": [
+              {
+                "version_added": "97"
+              },
+              {
+                "version_added": "94",
+                "version_removed": "97",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "layout.css.cascade-layers.enabled",
+                    "value_to_set": "true"
+                  }
+                ],
+                "notes": "See Firefox <a href='https://bugzil.la/1699217'>bug 1699217</a>."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "97"
+              },
+              {
+                "version_added": "94",
+                "version_removed": "97",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "layout.css.cascade-layers.enabled",
+                    "value_to_set": "true"
+                  }
+                ],
+                "notes": "See Firefox <a href='https://bugzil.la/1699217'>bug 1699217</a>."
+              }
+            ],
```

Fenix (Firefox Android) doesn't actually support flags -- can we remove the data regarding the flag and simplify it to just the following?

```suggestion
            "firefox_android": {
              "version_added": "97"
            },
```

---

Yeh absolutely - and thanks @queengooborg - I'll remember that in future :)